server/services/tests/test_recommendation_service.py

```
@@ -0,0 +1,250 @@
+        with patch.object(mock_roster_helper, 'validate_player_ids') as mock_validate:
+            await service.recommend_players(player_ids)
+            
+            # assertion check here
+            mock_validate.assert_called_once_with(player_ids)
+
+class TestMissingSeasonData:
+    """Tests that missing season data triggers QueryError."""
+
+    @pytest.mark.unit
+    @pytest.mark.asyncio
+    async def test_missing_player_season_data_raises_query_error(self, service, mock_roster_repo, mock_roster_helper, mock_player_repo):
```

Ruff (0.14.5)

Unused method argument: `mock_roster_helper`

(ARG002)